rust/otap-dataflow/crates/otap/src/syslog_cef_receiver.rs

```diff
@@ -349,6 +415,34 @@ impl local::Receiver<OtapPdata> for SyslogCefReceiver {
     }
 }
 
+/// RFC-aligned metrics for Syslog CEF receiver.
+///
+/// Key fields follow the component universal telemetry RFC for receivers:
+/// received.items.<signal>.<outcome>
+#[metric_set(name = "syslog_cef.receiver.metrics")]
+#[derive(Debug, Default, Clone)]
+pub struct SyslogCefReceiverMetrics {
+    /// Number of log records successfully received and forwarded
```

Comment:
This is not correct. This metric only tracks the number of logs forwarded. The received logs could be higher.

Reply:
added a dedicated pre-parse counter to get the correct received numbers